core/admin/mailu/api/common.py

```diff
@@ -5,13 +5,15 @@
 import hmac
 from functools import wraps
 from flask_restx import abort
+from sqlalchemy.sql.expression import label
 
-def fqdn_in_use(*names):
-    for name in names:
-        for model in models.Domain, models.Alternative, models.Relay:
-            if model.query.get(name):
-                return model
-    return None
+def fqdn_in_use(name):
+    d = models.db.session.query(label('name', models.Domain.name))
+    a = models.db.session.query(label('name', models.Alternative.name))
+    r = models.db.session.query(label('name', models.Relay.name))
+    if d.union_all(a).union_all(r).filter_by(name=name).count() > 0:
```

Comment:
Please use the .exists() variant as suggested in https://github.com/Mailu/Mailu/pull/2464#discussion_r1032557167